### PR TITLE
setting up project

### PR DESCRIPTION
**Pull request by pritishshah**

### CODE REVIEW

File: cmd/tokentransform/tokentransform.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"context"
```
Comment (mafarinacci) [inline cmd/tokentransform/tokentransform.go]: Imports should be broken up logically

import (
	goflag "flag"
	"fmt"
	"math/rand"
	"os"
	"time"

	"github.com/spf13/pflag"

	utilflag "k8s.io/apiserver/pkg/util/flag"
	"k8s.io/kubernetes/pkg/kubectl/cmd"
	"k8s.io/kubernetes/pkg/kubectl/util/logs"

	// Import to initialize client auth plugins.
	_ "k8s.io/client-go/plugin/pkg/client/auth"
)

File: cmd/tokentransform/tokentransform.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	Port int `envconfig:"default=7777"`
+}
+
+func main() {
+	if err := envconfig.Init(&conf); err != nil {
+		fmt.Printf("unable to initialize envconfig: %s", err)
+		return
+	}
+	
+	hostBuilder := strings.Builder{};
```
Comment (mafarinacci) [inline cmd/tokentransform/tokentransform.go]: semicolon?

File: Gopkg.toml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+# Exclude files. Helps to reduce size of vendor directory.
+[prune]
+  # prunes Go test files
+  go-tests = true
+  # prunes files that are not used by Go
+  non-go = true
+  # indicates that files from directories that do not appear in the package import graph should be pruned
+  unused-packages = true
+
+ [[constraint]]
```
[the remark this answers is not in the document]
Comment (pritishshah) [reply Gopkg.toml]: Totally agreed. I introduced version or revision in toml file, that will fix this issue.

File: cmd/tokentransform/tokentransform.go (hunk 1 above)
Comment (pritishshah) [reply cmd/tokentransform/tokentransform.go]: done

File: cmd/tokentransform/tokentransform.go [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+		fmt.Println("Starting GRPC server")
+		if err := grpcServer.Serve(grpcL); err != nil {
+			fmt.Printf("Unable to start external gRPC server")
+		}
+		
+	}()
+	
+	// start gateway server
+	go func() {
+		fmt.Println("Starting GRPC Gateway server")
```
[the remark this answers is not in the document]
Comment (pritishshah) [reply cmd/tokentransform/tokentransform.go]: Yup. Using logger now.

File: cmd/tokentransform/tokentransform.go (hunk 2 above)
Comment (pritishshah) [reply cmd/tokentransform/tokentransform.go]: Java Habit. Removed it.

**Pull request merged**